deepface-server/app/routes.py

```
@@ -0,0 +1,315 @@
+    content_type = request.headers.get("content-type", "")
+
+    if "multipart/form-data" in content_type:
+        form = await request.form()
+        extra = {k: v for k, v in form.items() if k != "file"}
+        if file is None:
+            raise _422("Provide a multipart 'file' field.")
+        image_bytes = await file.read()
+    else:
+        try:
+            payload = await request.json()
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)